### ISSUE TITLE
service enable not set for varnish

### ISSUE THREAD

**Issue by tobixen** [maxchk/puppet-varnish#140]
The current (master) manifests apparently sets this as default:

    service {'varnish':
      ensure => running;
    }

this will cause puppet to start up varnish if it's not already running - but it's not good enough, our new varnish VM did not pass the reboot test.  Ref https://docs.puppet.com/puppet/4.9/types/service.html enable should also be set:

    service {'varnish':
      ensure => running,
      enable => true;
    }

As for now I simply did a "systemctl enable varnish" on the server, it may be that I'll write up a pull request at a later stage if I get to it.

**Comment by kwisatz** [comment]
Please do, I ran into the same problem just now. I'm just not sure why it suddenly appeared with an upgrade from puppet 3.7.2 to 3.8.7, but has not gone away with a downgrade…

**Comment by kwisatz** [comment]
Nope, my problem doesn't seem to be related to this. I'll create a new ticket.

**Comment by func0der** [comment]
@maxchk Close this, pls. Too many open issues let the module look bad.

**Comment by tobixen** [comment]
I still believe this is a real issue.  The fix is very simple though.

**Comment by EmersonPrado** [comment]
Let's fix it. Added [PR 158](https://github.com/maxchk/puppet-varnish/pull/158).
